src/arrayMethodSort.js

```diff
@@ -4,9 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else {
+        return -1;
```

Comment:
The default `compareFunction` should return 0 when `a` and `b` are equal to mimic the behavior of the native sort. Currently, it will incorrectly order equal elements.